osmosis-1/osmosis.zone_assets.json

```diff
@@ -1611,6 +1611,13 @@
       "osmosis_main": true,
       "osmosis_frontier": true
     },
+    {
```

Comment:
please move this to the bottom of the file

Reply:
Done for both of them but still not getting the pool ID part. I read somewhere on the readme files that the ID had to be created by Osmosis themself. But to be Honest that part lost me a bit.

Reply:
When you created a pool, the transaction should tell you the ID for the pool that was created. If you have the transaction hash, you should be able to confirm it there.

Reply:
Got it. So basically now at this point it should be ready to be created? And if so, can it be done from the front end directly or CLI using osmosisd?

Reply:
Yes, that's right. And either method works. If you want a preview frontend so that the token doesn't appear as a generic '?' ibc/HASH asset, then I'll need to whip up a branch first to deploy it.

Reply:
Perfect we will take care of this tomorrow and let you know about the branch if needed. Thank you so much Jeremy.

Reply:
Hi Jeremy and good afternoon. My team mate tried to create the pool on the front end but couldn't find OIN in there, Can i gently ask for a stage link or something to help us track the asset. Thanks

Reply:
Here is a preview link you can use the replaces the ibc/hash with symbol OIN:
https://osmosis-frontend-git-jp69-asset-oin-add-osmo-labs.vercel.app/assets

Reply:
Thank you so much Jeremy. I'll post the pool ID as soon as the pool is up